src/components/CountryBoard.vue

```diff
@@ -1,7 +1,119 @@
-<script lang="ts" setup></script>
+<script lang="ts" setup>
+import { useCountriesStore } from "@/stores/countries.store";
+import { computed, onMounted } from "vue";
+import { ICountry } from "@/types/countries";
+
+const store = useCountriesStore();
+const isLoading = computed((): boolean => store.$state.isLoading);
+const countriesList = computed((): ICountry[] => store.$state.countriesList);
+const countryBasicInfoTitles = [
+  {
+    title: "Population",
+    key: "population",
+  },
+  {
+    title: "Region",
+    key: "region",
+  },
+  {
+    title: "Capital",
+    key: "capital",
+  },
+];
+
+onMounted(() => {
+  store.getCountriesList();
+});
+</script>
 
 <template>
-  <div>Country</div>
+  <div class="c-countryBoard">
+    <div v-if="!isLoading" class="c-countryBoard__board">
+      <div
+        v-for="(country, index) in countriesList"
+        :key="`${country}-${index}`"
```

Comment:
to nie zadziła za dobrze, bo country jest obiektem - dla przykładu pierwszy klucz to [Object object]-0